pandas/core/window/rolling.py

```diff
@@ -1822,37 +1829,16 @@ def _get_corr(a, b):
 
 
 class Rolling(RollingAndExpandingMixin):
-    @cache_readonly
-    def is_datetimelike(self) -> bool:
```

Comment:
Note that those attributes *are* publc (whether we like that or not ..), and so should ideally be deprecated first if we remove them.

Reply:
(note I was looking at some rolling-related PRs because of a dask failure, see https://github.com/pandas-dev/pandas/pull/38641#issuecomment-754070881, but this change about `is_datetimelike` didn't actually cause an issue in dask, to be clear)

Reply:
I am -1 on deprecating anything here. these are internal and never documented as public.